include/DREAM/OtherQuantityHandler.hpp

```diff
@@ -82,8 +83,10 @@ namespace DREAM {
 			// List of kinetic ionization rates for each ion species
 			std::vector<IonKineticIonizationTerm*> f_hot_kin_rates;
 			std::vector<IonKineticIonizationTerm*> f_re_kin_rates;
+            // List of approximated RE impact ionization rates for teach ion species
+            std::vector<IonFluidRunawayIonizationTerm*> f_re_kin_approx_rates;
```

Comment:
Isn't it confusing if this variable has ``f_re`` in the name, even though it has nothing to do with ``f_re``? I would propose a name such as ``n_re_kin_rates`` instead.